CHANGES/README.rst

```diff
@@ -34,21 +31,22 @@ for the users to understand what it means.
 combined with others, it will be a part of the "news digest"
 telling the readers **what changed** in a specific version of
 the library *since the previous version*. You should also use
-reStructuredText syntax for highlighting code (inline or block),
-linking parts of the docs or external sites.
-If you wish to sign your change, feel free to add ``-- by
+*reStructuredText* syntax for highlighting code (inline or block),
+linking parts of the docs or external sites. However, you do not need to
+reference the issue or PR numbers here as *towncrier* will automatically
+add a reference to all of the affected issues when rendering the news
+file. If you wish to sign your change, feel free to add ``-- by
 :user:`github-username``` at the end (replace ``github-username``
 with your own!).
 
 Finally, name your file following the convention that Towncrier
 understands: it should start with the number of an issue or a
 PR followed by a dot, then add a patch type, like ``feature``,
-``doc``, ``misc`` etc., and add ``.rst`` as a suffix. If you
-need to add more than one fragment, you may add an optional
-sequence number (delimited with another period) between the type
-and the suffix.
+``doc``, ``misc`` etc. If you need to add more than one fragment,
+you may add an optional sequence number (delimited with another period)
+between the type and the suffix.
 
-In general the name will follow ``<pr_number>.<category>.rst`` pattern,
```

Comment:
Plz keep `.rst`. While it's not mandatory, it makes it easier for people to understand what syntax is inside.

Reply:
We don't have a single file in CHANGES with .rst currently, and the other instructions didn't have this...

Reply:
True. But the common template I'm syncing over does. There's also `.rst` files in yarl/frozenlist/multidict. We just need to do a better job of consolidating those bits.